projects/appearances/protos/Sand.proto

```diff
@@ -0,0 +1,36 @@
+#VRML_SIM R2019b utf8
+# license: Apache License 2.0
+# license url: http://www.apache.org/licenses/LICENSE-2.0
+# A sand material.
+
+PROTO Sand [
```

Comment:
I think we should merge `SandyGround` in this PROTO (in another PR, in the develop branch) thanks to a new  `Sand.type` field.